pages/contact-old-age-security.js

```diff
@@ -0,0 +1,175 @@
+import PropTypes from 'prop-types'
+import { Heading, TableContent } from '@dts-stn/service-canada-design-system'
+import { Fragment } from 'react'
+import en from '../locales/en'
+import fr from '../locales/fr'
+import ContactSection from '../components/contact/ContactSection'
+import ContactProvince from '../components/contact/ContactProvince'
+import { getBetaBannerContent } from '../graphql/mappers/beta-banner-opt-out'
+import { getBetaPopupExitContent } from '../graphql/mappers/beta-popup-exit'
+import { getBetaPopupNotAvailableContent } from '../graphql/mappers/beta-popup-page-not-available'
+import { getContactOldAgeSecurityContent } from '../graphql/mappers/contact-old-age-security'
+import logger from '../lib/logger'
+import Modal from 'react-modal'
+import React from 'react'
+import ExitBetaModal from '../components/ExitBetaModal'
+
+export default function ContactOldAgeSecurity(props) {
+  /* istanbul ignore next */
+  const t = props.locale === 'en' ? en : fr
+
+  const [openModalWithLink, setOpenModalWithLink] = React.useState({
+    isOpen: false,
+    activeLink: '/',
+  })
+
+  function openModal(link) {
+    setOpenModalWithLink({ isOpen: true, activeLink: link })
+  }
+
+  function closeModal() {
+    setOpenModalWithLink({ isOpen: false, activeLink: '/' })
+  }
+
+  return (
+    <div
+      id="homeContent"
+      data-testid="homeContent-test"
+      data-cy="oasContactUsContent"
+    >
+      <Heading id="my-dashboard-heading" title={props.pageContent.title} />
+      <div className="py-5" />
+      <TableContent
+        sectionList={props.pageContent.items.map((item, i) => {
+          return { name: item.title, link: `#${item.id}` }
+        })}
+      />
+
+      {props.pageContent.items.map((item, i) => (
+        <Fragment key={i}>
+          {item.layout === 'provinces' ? (
+            <ContactProvince {...item} i={i} />
+          ) : (
+            <ContactSection programUniqueId={i} {...item} />
+          )}
+        </Fragment>
+      ))}
+
+      <Modal
+        className="flex justify-center bg-black/75 h-full"
+        isOpen={openModalWithLink.isOpen}
+        onRequestClose={closeModal}
+        contentLabel={t.aria_exit_beta_modal}
+      >
+        <ExitBetaModal
+          closeModal={closeModal}
+          closeModalAria={t.close_modal}
+          continueLink={openModalWithLink.activeLink}
+          popupId={props.popupContent.popupId}
+          popupTitle={props.popupContent.popupTitle}
+          popupDescription={props.popupContent.popupDescription}
+          popupPrimaryBtn={props.popupContent.popupPrimaryBtn}
+          popupSecondaryBtn={props.popupContent.popupSecondaryBtn}
+        />
+      </Modal>
```

Comment:
The modal is never used. Do we want to keep this code?

Reply:
Same thing for the modal functions (open and close)

Reply:
Removed modals from all contact pages, good catch!